packages/events/EventPluginHub.js

```diff
@@ -163,17 +163,27 @@ function extractEvents(
   targetInst: null | Fiber,
   nativeEvent: AnyNativeEvent,
   nativeEventTarget: EventTarget,
+  passive: null | boolean,
```

Comment:
Can we avoid null | boolean? I imagine this would cause deopts.
For example, it could be an enum (0, 1, 2).